### PR TITLE
Update dependebot recommendations as of 07-16-2021

### PR DESCRIPTION
**Pull request by Oceas**
Ticket: https://webdevstudios.atlassian.net/browse/CC-257

Update all NPM packages per dependabot recommendations as of 07/16/2021

### CODE REVIEW

**Comment by richaber** [comment]
`lockfileVersion` changed from `1` to `2`.
What version of node and npm were used to generate this?

File: package.json [hunk 1]
```diff
@@ -4,14 +4,19 @@
   "description": "",
   "main": "index.js",
   "dependencies": {
-    "filter-validate-email": "^1.0.4"
+    "browserslist": "^4.16.6",
+    "elliptic": "^6.5.4",
+    "filter-validate-email": "^1.0.4",
+    "ini": "^1.3.7",
+    "ssri": "^6.0.2",
+    "y18n": "^4.0.1"
```
Comment (richaber) [inline package.json:12]: Presumably, all of these _new_ `dependencies` except `filter-validate-email` should be in `devDependencies`, as the only actual production dependency prior to this PR was `filter-validate-email`.
Presumably browserslist, elliptic, ini, ssri, and y18n are devDependencies or peer dependencies of the build tools listed in our own devDependencies.

**Review by richaber — CHANGES_REQUESTED** [review]

File: package.json (hunk 1 above)
Comment (Oceas) [reply package.json]: Fair point. Changes made.

**Comment by richaber** [comment]
Tested `npm install` and `npm run build` with Node v 14.17.5 and NPM v 7.22.0.
Build output appears successful.
Tested resulting bundle.js file on checkout and it appears to function as expected, adding my email address to the abandoned checkout row in MySQL.
```
mysql> SELECT checkout_id, user_id, user_email, checkout_updated, checkout_created, checkout_uuid FROM local.wp_cc_abandoned_checkouts;
+-------------+---------+------------------+---------------------+---------------------+--------------------------------------+
| checkout_id | user_id | user_email       | checkout_updated    | checkout_created    | checkout_uuid                        |
+-------------+---------+------------------+---------------------+---------------------+--------------------------------------+
|         100 |       0 | test@example.com | 2021-09-09 22:24:28 | 2021-09-09 22:24:28 | 3040c2ed-4ed8-4f48-812d-3a56f1a5e1fd |
+-------------+---------+------------------+---------------------+---------------------+--------------------------------------+
1 row in set (0.00 sec)
```

**Review by richaber — APPROVED** [review]

**Pull request merged**